src/OptionSettlementEngine.sol

```diff
@@ -43,29 +43,17 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
     // Internal Data Structures
     //////////////////////////////////////////////////////////////*/
 
-    /// @notice Store the exercise state of a given collateral bucket
+    /// @notice Stores the exercise state of a given collateral bucket.
     enum BucketExerciseState {
         Exercised,
         PartiallyExercised,
         Unexercised
     }
 
     /**
-     * @notice Claims can be used to write multiple times. This struct is used to keep track
-     * of how many options are written against a claim in each bucket, in order to
-     * correctly perform fair exercise assignment.
-     */
-    struct ClaimIndex {
-        /// @custom:member amountWritten The amount of option contracts written into claim for given bucket.
-        uint112 amountWritten;
-        /// @custom:member bucketIndex The index of the Bucket into which the options collateral was deposited.
-        uint96 bucketIndex;
-    }
-
-    /**
-     * @notice Represents the total amount of options written and exercised for a group of
-     * claims bucketed. Used in fair assignment to calculate the ratio of
-     * underlying to exercise assets to be transferred to claimants.
+     * @notice Represents the total amount of options written and exercised for a 
+     * group of claims bucketed. Used in fair assignment to calculate the ratio of
```

Comment:
group of claims bucket is weird verbiage. Maybe "Stores the state of options written and exercised for a bucket".

Reply:
Cleaned up